app/models/location.rb

```diff
@@ -49,6 +49,7 @@ class Location < ApplicationRecord
 
   scope :incomplete, lambda {
     where.not(confirmed: true)
+      .or(where(confirmed: nil))
```

Comment:
Wouldn't nil not be true tho?

Reply:
Yeah that's what I thought, this is how the bug didn't get caught earlier - ActiveRecord's where.not translates to a sql != which doesn't return null values - see https://thoughtbot.com/blog/activerecord-s-where-not-and-nil

Reply:
Oh yeah, okay. Good spot!